lib/cartopy/crs.py

```diff
@@ -1652,15 +1654,16 @@ class Geostationary(_Satellite):
 
     """
     def __init__(self, central_longitude=0.0, satellite_height=35785831,
-                 false_easting=0, false_northing=0, globe=None):
+                 false_easting=0, false_northing=0, globe=None, sweep_axis='y'):
```

Comment:
Should the default be `'y'` or `None`? I'm a bit confused about this I must admit. The document you linked to suggests the proj4 default is `'y'`, so is that what we have been getting anyway since proj4.8? In other words, is using `'y'` as the default backwards compatible?

The last paragraph of https://github.com/OSGeo/proj.4/wiki/Geostationary appears to contradict the previous 2, suggesting meteosat uses sweep `'x'` and GOES uses sweep `'y'`, which doesn't help my undertanding!